### ISSUE TITLE
Modelerfour doesn't parse TextAnalytics 3.0preview

### ISSUE THREAD

**Issue by lmazuel** [Azure/autorest.modelerfour#72]
> FATAL: Error: Unrecognized schema type:'undefined' / format: undefined {"x-ms-metadata":{"apiVersions":["v3.0-preview.1"],"filename":["mem:///98?oai3.shaken.json"],"name":"DocumentSentiment-documentScores","originalLocations":["https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-preview.1/TextAnalytics.json#/components/schemas/DocumentSentiment-documentScores"]},"description":"Document level sentiment confidence scores between 0 and 1 for each sentiment class."}
>   Error: Plugin modelerfour reported failure.

https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-preview.1/TextAnalytics.json

**Comment by lmazuel** [comment]
Autorest v2 was fine, we just merged a preview SDK using autorest.

**Comment by fearthecowboy** [comment]
O_o

What did you generate for the property:

``` 
        "documentScores": {
          "description": "Document level sentiment confidence scores between 0 and 1 for each sentiment class."
        },
```

It looks like that's only used in a response, but still, strongly-typed languages would have to generate 'object' as the type; and it wouldn't be very convenient during consumption.

Given the property, I'm assuming that this should be a `type: Number`. 

I'm thinking we should fail earlier on fields like this since they offer nothing to the consumer but a miserable experience. 

Thoughts?

**Comment by fearthecowboy** [comment]
Will be addressed by #73

**Comment by fearthecowboy** [comment]
Fixed in #73  -- install via `autorest-beta --use:@autorest/modelerfour@4.2.83`

**Comment by iscai-msft** [comment]
Not crashing
